### PR TITLE
Feature/uart print

### PR DESCRIPTION
**Pull request by NicolaiFransen**

### CODE REVIEW

File: Software/Development/SemesterProject/System/scheduler.c [hunk 1]
```diff
@@ -113,8 +119,13 @@ void task50us(void)
     performErrorMonitoring();
 }
 
+void task1ms(void)
+{
```
Comment (murgui) [inline Software/Development/SemesterProject/System/scheduler.c:92]: Nice Empty task

File: Software/Development/SemesterProject/System/scheduler.c [hunk 2]
```diff
@@ -215,3 +226,17 @@ Uint32 getSystemClock(void)
 {
     return sysClock;
 }
+
+void printUART(void)
+{
+//    Print stuff with UART here�!
+//    UARTIntPrint("Vc ", (int)getControlsupplyMeasurement() * 100);
+    UARTIntPrint("Vdc ", (int)(getDCLinkMeasurement() * 100));
```
Comment (murgui) [inline Software/Development/SemesterProject/System/scheduler.c]: maybe everything should be commented?

**Review by murgui — CHANGES_REQUESTED** [review]
Good.

**Review by murgui — APPROVED** [review]
Go for it (if it is not crashing too much)

File: Software/Development/SemesterProject/System/Include/scheduler.h [hunk 3]
```diff
@@ -48,6 +48,7 @@ void runTask(void (*functionPTR)());
 
 void task50us(void);
 void task100us(void);
+void task1ms(void);
```
Comment (AitorTeran) [inline Software/Development/SemesterProject/System/Include/scheduler.h]: Maybe 1ms is too much, during tests, we were printing at 10ms and it was crashing (true, with the long names but still....)

**Comment by murgui** [comment]
I think that in addition to have the task, a short description including the character limitation should be added. So for example saying it was tested that with 50 characters printed in the function, it was not crashing.

File: Software/Development/SemesterProject/System/Include/scheduler.h (hunk 3 above)
Comment (AitorTeran) [reply Software/Development/SemesterProject/System/Include/scheduler.h]: An idea is to call the task something like taskUART(void) and then set the cyclicity to a constant that we can change depending on the case...

File: Software/Development/SemesterProject/System/scheduler.c [hunk 4]
```diff
@@ -113,8 +119,14 @@ void task50us(void)
...
 }
 
+void task1ms(void)
+{
+    printUART();
+}
+
 void task10ms(void)
 {
+    printUART();
```
Comment (AitorTeran) [inline Software/Development/SemesterProject/System/scheduler.c]: so printing uart both every 1ms and 10ms?

**Review by AitorTeran — COMMENTED** [review]
mmmm

**Pull request merged**